spec/error_spec.rb

```diff
@@ -4,13 +4,14 @@ module CoverMyMeds
   module Error
     describe HTTPError do
 
-      subject { described_class.new status, error_json, http_method, rest_resource }
+      subject { described_class.new status, error_json, http_method, url }
 
       specify "#message" do
-        expect(subject.message).to eq(
```

Comment:
All this did was reproduce the exact implementation.  That is kind of like testing that a == a, and really shouldn't ever fail, regardless of how we *want* the output to look.